### PR TITLE
PI-764: Adding back role to cf-component-loading to be consistent wit…

### PR DESCRIPTION
**Pull request by jwineman**

### CODE REVIEW

**Comment by jwineman** [comment]
@terinjokes @toekneestuck @wyuenho @marksteyn @thellimist

**Comment by wyuenho** [comment]
LGTM

File: packages/cf-component-icon/test/Icon.js [hunk 1]
```diff
@@ -40,4 +40,12 @@ describe('Icon', function() {
       <i className="cf-icon cf-icon--info-sign"></i>
     );
   });
+
+    it('should render role', function() {
+    assertEqualJSX(
```
Comment (thejameskyle) [inline packages/cf-component-icon/test/Icon.js]: What even is indentation

Comment (jwineman) [inline packages/cf-component-icon/test/Icon.js]: lol I'll fix.  Had to edit in sublime cause intellij kept crashing for some reason.  Need you to add me as a publisher to all the packages though.

Comment (terinjokes) [inline packages/cf-component-icon/test/Icon.js]: @thejameskyle indentation adds so many wasted bytes to a git clone. if we remove all indentation cloning will be much faster.

**Pull request merged**